tests/src/kernel/hipLaunchParm.cpp

```diff
@@ -23,75 +23,768 @@ THE SOFTWARE.
  * HIT_END
  */
 
+#include <cstdint>
 #include "hip/hip_runtime.h"
 #include "test_common.h"
-#include "hip/hip_runtime_api.h"
-#include <iostream>
 
-__global__ void vAdd(hipLaunchParm lp, float* a) {}
+// Memory alignment is broken
+#define ENABLE_ALIGNMENT_TEST 0
```

Comment:
Are the failed tests fail with both hcc and hip-clang?
If not, you can use `#if __HCC__` to differentiate them. Turn off the the test for only hcc or hip-clang.

Reply:
Yes, failing on both hcc and hip-clang?